server/src/main/scala/drt/server/feeds/edi/EdiFeed.scala

```diff
@@ -140,7 +143,8 @@ case class EdiFeed(ediClient: EdiClient)
         CarrierScheduled = None,
         ApiPax = None,
         ScheduledDeparture = None,
-        RedListPax = None
+        RedListPax = None,
+        TotalPax =  SortedSet(TotalPaxSource(flight.Passengers.getOrElse(0),feedSource,None))
```

Comment:
We reduce code by setting TotalPax in the ArrivalsGraphStage rather than in each individual feed. We also then protect ourselves from setting the wrong feed source if we reuse a feed for a different type